### PR TITLE
account activation & reset password

### PR DESCRIPTION
**Pull request by AilinSoon**

### CODE REVIEW

**Comment by AilinSoon** [comment]
ready

File: app/assets/stylesheets/account_activations.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
```
Comment (huongnt-2545) [inline app/assets/stylesheets/account_activations.scss]: delete this file

File: app/assets/stylesheets/password_resets.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the PasswordResets controller here.
```
Comment (huongnt-2545) [inline app/assets/stylesheets/password_resets.scss]: delete this file

File: app/helpers/account_activations_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (huongnt-2545) [inline app/helpers/account_activations_helper.rb]: delete this file

File: app/helpers/password_resets_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module PasswordResetsHelper
```
Comment (huongnt-2545) [inline app/helpers/password_resets_helper.rb]: delete this file

File: app/views/shared/_error_messages.html.erb [hunk 5]
```diff
@@ -1,11 +1,11 @@
-<% if object.errors.any? %>
```
Comment (huongnt-2545) [inline app/views/shared/_error_messages.html.erb]: keep this, don't change in this file

File: app/views/password_resets/edit.html.erb [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<% provide(:title, t(:reset)) %>
+<h1><%= t(:reset) %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user, url: password_reset_path(params[:id])) do |f| %>
+      <%= render 'shared/error_messages' %>
```
Comment (huongnt-2545) [inline app/views/password_resets/edit.html.erb]:
```suggestion
      <%= render 'shared/error_messages', object: f.object %>
```

File: config/application.rb [hunk 7]
```diff
@@ -6,6 +6,8 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+private
```
Comment (huongnt-2545) [inline config/application.rb]: remove this line

File: test/controllers/account_activations_controller_test.rb [hunk 8]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (huongnt-2545) [inline test/controllers/account_activations_controller_test.rb]: delete this file

File: test/controllers/password_resets_controller_test.rb [hunk 9]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```
Comment (huongnt-2545) [inline test/controllers/password_resets_controller_test.rb]: delete this file

File: test/mailers/previews/user_mailer_preview.rb [hunk 10]
```diff
@@ -0,0 +1,14 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
```
Comment (huongnt-2545) [inline test/mailers/previews/user_mailer_preview.rb]: delete this file

File: test/mailers/user_mailer_test.rb [hunk 11]
```diff
@@ -0,0 +1,20 @@
+require "test_helper"
```
Comment (huongnt-2545) [inline test/mailers/user_mailer_test.rb]: delete this file

**Comment by AilinSoon** [comment]
ready

**Pull request merged**